src/bentoml_cli/deployment.py

```diff
@@ -620,6 +620,9 @@ def create_deployment(
         spinner.log_progress.add_task(
             f"[bold green]Successfully created deployment '{deployment.name}' in cluster '{deployment.cluster}'[/bold green]"
         )
+        spinner.log_progress.add_task(
+            f"[bold blue]To check the deployment details, go to: {deployment.admin_console}[/bold blue]"
```

Comment:
lol I know why we saw `old green` yesterday

Reply:
A small tip is to use `[/]` as the closing tag to avoid mistakes.